server/routes/journeys/record-review/participants-summary/view.njk

```diff
@@ -0,0 +1,118 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends "partials/layout.njk" %}
+
+{% set fullWidth = attendees.length %}
+{% set pageTitle = "Participant and contribution details - Record a CSIP review" %}
+
+{% block innerContent %}
+    {% include 'base/components/review-caption.njk' %}
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-7">Log participants and contributions to the review</h1>
+
+    {% for attendee in attendees %}
+        {{ govukSummaryList({
+            card: {
+                title: { text: "Participant: " + attendee.name },
+                actions: {
+                    items: [
+                        {
+                            href: 'delete-participant/' + attendee.attendeeUuid,
+                            text: "Delete",
+                            visuallyHiddenText: "the information about the participant (" + attendee.name + ")",
```

Comment:
the card title is automatically appended to the visually hidden text.
ie, ` (Participants: Dave Jones)` will be auto appended, we probably don't need to repeat it in each piece of hidden text.

Reply:
Oh? Does the component do that itself?

Reply:
yea it's magic by the govuk component